app/components/catalog-settings/template.hbs

```diff
@@ -0,0 +1,12 @@
+<h4>Catalog</h4>
+<hr>
+<h5>You can add and remove catalogs here. When adding ensure you give your catalog a name and a git repo to pull from.</h5>
```

Comment:
The value can be anything `git clone` will take, which includes things like

- https://github.com/user/repo
- git@github.com:user/repo
- /path/to/local/folder